affiliated/index.html

```diff
@@ -164,105 +165,72 @@ <h1 id="coordinated-package-list">Coordinated Packages<a class="paralink" href="
-	  <p>The following table lists all currently registered affiliated packages. They are determined from the <a href="http://www.astropy.org/affiliated/registry.json">json file</a>, which is the actual authoritative registry.</p>
-<!--	  <h3><u>Affiliated Packages</u></h3>-->
-      <p>Total number of affiliated packages: <strong id="total-affiliated-pkgs"></strong></p>
+	  <p>The following table lists all currently registered affiliated packages.</p>
+    <p>NOTE: The listing is current empty because Astropy has just accepted
+    <a href="https://github.com/astropy/astropy-APEs/blob/main/APE22.rst">APE 22</a>
+    in January 2024. We are in the midst of transitioning to the new process in
+    partnership with pyOpenSci, so we really appreciate your patience.
+    Pre-APE 22 legacy listing is available <a href="legacy.html">here</a>.
+    </p>
 
-	<table border="1" class="docutils" id="affiliated-package-table">
-	<colgroup>
-	<col width="5%" />
-	<col width="3%" />
-	<col width="3%" />
-	<col/>
-	</colgroup>
-	<thead valign="bottom">
-	<tr class="row-odd">
-	<th class="head"></th>
-	<th class="head"></th>
-	<th class="head"></th>
-	<th class="head"></th>
-	</tr>
-	</thead>
-	<tbody valign="top">
-	<tr class="row-even"><td rowspan="1">Loading...</td>
-	<td rowspan="1">&nbsp;</td>
-	<td rowspan="1">&nbsp;</td>
-	<td rowspan="1">&nbsp;</td>
-	<td rowspan="1">&nbsp;</td>
-	</tr>
-	<tr class="row-odd">
-	<td colspan="1">&nbsp;</td>
-	<td colspan="3">&nbsp;</td>
-	</tr>
-	</tbody>
-	</table>
+<!-- TODO: Leah Wasser is going to give Erik Tollerud some RSS/XML to ingest here
+(maybe https://github.com/pyOpenSci/pyopensci.github.io/blob/main/_data/packages.yml).
+We have promised pyOpenSci that we would also link back to their canonical listing
+here, with their Astropy filter applied. -->
 
 	</section>
 
 	<section id="affiliated-instructions">
 
 	<h1 id="becoming-an-affiliated-package">Becoming an Affiliated Package<a class="paralink" href="#becoming-an-affiliated-package" title="Permalink to this headline">¶</a></h1>
 
-	<p>If you are a developer of an astronomy package and would like your package
-	to become affiliated with the Astropy Project, please take a look at the
-	<a href="https://github.com/astropy/astropy-project/blob/main/affiliated/affiliated_package_review_process.md#proposing-an-affiliated-package">
-  instructions for proposing an affiliated package</a>. We recommend that you
-	also take a look at the
+  <p>Astropy <a href="https://www.pyopensci.org/partners.html">leverages the pyOpenSci peer review process</a>
+  to vet affiliated packages. If you are a developer of an astronomy package and would like your package to become
+  affiliated with the Astropy Project, you can submit your package directly to pyOpenSci and be accepted by pyOpenSci
+  and become affiliated with astropy through a single review process. You can also opt to be fast tracked through
```

Comment:
```suggestion
  affiliated with the Astropy Project, you submit your package directly to pyOpenSci.  
  Astropy will be involved in the review, which will allow your package to become an affiliated package through that review process. You can also opt to be fast tracked through
```
I think this clarifies better that it's a partnership, and this is the *only* way to get affiliated status now.